src/Codata/Musical/Colist/Base.agda

```diff
@@ -0,0 +1,66 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Coinductive lists: base type and functions
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --sized-types --guardedness #-}
+
+module Codata.Musical.Colist.Base where
+
+open import Codata.Musical.Notation
+open import Codata.Musical.Conat.Base using (Coℕ; zero; suc)
+open import Data.Bool.Base using (Bool; true; false)
+open import Data.List.Base using (List; []; _∷_)
+open import Data.Maybe.Base using (Maybe; nothing; just)
+open import Data.Nat.Base using (ℕ; zero; suc)
+
+------------------------------------------------------------------------
```

Comment:
As this is a new module, it would greatly benefit from a couple of variables for `a`, `A` and `B`?

Reply:
I used this as a pretext to hack away at `Codata.Musical.Colist` until not much was left.